spec/data/sample_manifest_excel/manifest_types.yml

```diff
@@ -188,10 +188,28 @@ plate_bioscan:
     - :date_of_sample_collection
     - :sample_taxon_id
     - :sample_common_name
-    - :bioscan_sample_description
+    - :sample_description_specimen_plate_barcode
     - :sample_type
     - :donor_id
     - :bioscan_control_type
+plate_anospp:
+  heading: "ANOSPP Plate"
+  asset_type: "plate"
+  columns:
+    - :sanger_plate_id
+    - :well
+    - :sanger_sample_id
+    - :supplier_name
+    - :retention_instruction
+    - :cohort
+    - :country_of_origin
+    - :date_of_sample_collection
+    - :sample_taxon_id
+    - :sample_common_name
+    - :sample_description_specimen_plate_barcode
```

Comment:
Why does ANOSPP needs specimen plate barcode ?

Reply:
Requriements are similar to bioscan, they want that second label with the specimen plate barcode

Reply:
I asked because Bioscan uses it for control sample naming later and there are no control samples for ANOSPP. If it is in requirements, that is OK.